src/services/listCourtCases.ts

```diff
@@ -130,11 +130,15 @@ const listCourtCases = async (
   if (reasonCode) {
     query.andWhere(
       new Brackets((qb) => {
-        qb.where("trigger.trigger_code ilike '%' || :reason || '%'", {
-          reason: reasonCode
-        }).orWhere("courtCase.error_report ilike '%' || :reason || '%'", {
-          reason: reasonCode
+        qb.where("trigger.trigger_code ilike :trigger", {
+          trigger: reasonCode
         })
+          .orWhere("courtCase.error_report ilike :firstException || '%'", {
+            firstException: reasonCode + "||"
+          })
+          .orWhere("courtCase.error_report ilike '%' || :exception || '%'", {
+            exception: " " + reasonCode + "||"
+          })
```

Comment:
Could you use template literals, Eg
```typescript
const string = `${var} hello`
```

Trying to read this is hard.

Reply:
Changed into template literals.